src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpGatewayController.cs

```diff
@@ -75,19 +76,15 @@ public async Task<IActionResult> ClosedApplications(int page = 1)
         [HttpGet("/Roatp/Gateway/{applicationId}")]
         public async Task<IActionResult> ViewApplication(Guid applicationId)
         {
-            var application = await _applyApiClient.GetApplication(applicationId);
-            if (application is null)
+            var vm = await _mediator.Send(new GetApplicationOverviewRequest(applicationId));
```

Comment:
Variable naming could be more descriptive

Reply:
why is it a viewmodel and not an application?

Reply:
Also... there's mediators in Web. Should be using ApiClients

Reply:
The handler returns viewmodel, not application

Reply:
The decision to use Handler, instead of APIs or services was taken for the whole Gateway

Reply:
@David-B-Read vm is used in all actions in this controller, so for consistency I will keep it for now.